From 1a90a0682ce807a8f76ff12ff373f57117cfca07 Mon Sep 17 00:00:00 2001
From: Martin Hrdlicka <martin.hrdlicka@nxp.com>
Date: Wed, 1 Mar 2023 10:46:37 +0100
Subject: [PATCH 59/63] arm64: dts: s32g-pfe: extend enum for
 nxp,pfeng-linked-phyif property

Extend nxp,pfeng-linked-phyif ethernet property enum to support
also HIF channels.

Issue: AAVB-6542
Upstream-Status: Pending 

Signed-off-by: Martin Hrdlicka <martin.hrdlicka@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/dt-bindings/net/s32g-pfe.h | 13 +++++++++----
 1 file changed, 9 insertions(+), 4 deletions(-)

diff --git a/include/dt-bindings/net/s32g-pfe.h b/include/dt-bindings/net/s32g-pfe.h
index 7350b4588dac..cbb1c9497c5e 100644
--- a/include/dt-bindings/net/s32g-pfe.h
+++ b/include/dt-bindings/net/s32g-pfe.h
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: GPL-2.0 */
 /*
- * Copyright 2021-2022 NXP
+ * Copyright 2021-2023 NXP
  *
  */
 
@@ -13,8 +13,13 @@
 #define PFE_HIF_CHANNEL_3		3
 #define PFE_HIF_CHANNEL_NOCPY		16
 
-#define PFE_EMAC_0			0
-#define PFE_EMAC_1			1
-#define PFE_EMAC_2			2
+#define PFE_PHYIF_EMAC_0		0
+#define PFE_PHYIF_EMAC_1		1
+#define PFE_PHYIF_EMAC_2		2
+#define PFE_PHYIF_HIF_NOCPY		4
+#define PFE_PHYIF_HIF_0			6
+#define PFE_PHYIF_HIF_1			7
+#define PFE_PHYIF_HIF_2			8
+#define PFE_PHYIF_HIF_3			9
 
 #endif /* __DT_BINDINGS_NET_S32G_PFE_H__ */
-- 
2.25.1

